references/displayed_attributes.md

```diff
@@ -24,12 +24,9 @@ Get the [displayed attributes](/guides/advanced_guides/settings.md#displayed-att
 | ------------- | ------------- |
 | **index_uid** | The index UID |
 
-### Example
+#### Example
```

Comment:
Why this one and not the others? (searchable attributes, ranking rules...)

Reply:
I realised it was not consistent everywhere in the doc. I will make an issue about it. I have fixed it at some places but not all